### ISSUE TITLE
Heello web - main timeline "shrinks" on ping deletion

### ISSUE THREAD

**Issue by katanacrimson** [Heello/Issues#9]
As on tin. Deleting pings that are currently on your main timeline will "shrink" it, and the timeline does not backfill with older content, giving the appearance of activity being dead.  Probably bad for encouraging people to be active, also means that content users may want to see (and would have seen prior to the original content being deleted having been posted) would no longer be visible.  

To put it another way - shifting the oldest timeline content is occurring (likely intentionally), but on ping deletion, _un_shifting timeline content back onto the stack isn't occurring.

**Comment by katanacrimson** [comment]
After going through this a few times now debugging another issue and deleting created tweets...

![http://i.imgur.com/3AUac5m.png](http://i.imgur.com/3AUac5m.png)

I guarantee you that account has tweets to display.  Showing that empty timeline on web is bad.
